packages/govuk-frontend/src/govuk/components/summary-list/summary-list.yaml

```
@@ -45,12 +45,12 @@ params:
       - name: actions
         type: object
         required: false
-        description: The actions of each row.
+        description: The action link content for each row item in the summary list component.
         params:
           - name: items
             type: array
             required: false
-            description: Array of action item objects.
+            description: The items within the summary list component.
```

This isn't quite right – this is the list of links to add to the row.

---

Good spot, I'll fix this one

```suggestion
            description: The action link items within the row item of the summary list component.
```